### PR TITLE
[PL-56710]: SMP 0.20.0 RN

### PR DESCRIPTION
**Pull request by GokulBansal0**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: SMP 0.20.0 RN
* Jira/GitHub Issue numbers (if any): [PL-56710](https://harness.atlassian.net/browse/PL-56710)
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

[PL-56710]: https://harness.atlassian.net/browse/PL-56710?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -123,6 +123,231 @@ Upon providing your credentials and the release version, the script will proceed
 
 :::
 
+## September 3, 2024, version 0.20.0
```
Comment (bharatgoelharness) [inline release-notes/self-managed-enterprise-edition.md:126]: sep 2

File: release-notes/self-managed-enterprise-edition.md [hunk 2]
```diff
@@ -123,6 +123,231 @@ Upon providing your credentials and the release version, the script will proceed
...
+
+- Fixed an issue where pipelines were getting queued when running concurrently. The fix ensures that the flush API log lines are sanitized to be less than 4MB, avoiding grpc `ResourceExhausted` failures. (CI-12879, ZD-64595)
+
+- Fixed an issue where, if the base image connector is overridden, the Docker build step does not work. With this fix, Docker-related images now properly gain privilege if the default connector is overridden. `buildx` images are now located [here]  (https://hub.docker.com/search?q=plugins%2Fbuildx). These images are added to the auto-privilege mode. Without this privilege, the image does not run. (CI-12583)
+
+- CI - Getting Started Page Visibility: Resolved an issue where the "Getting Started" page for CI was not visible to users without account-level edit permissions. (CI-12510)
+
+- Fixed issues where the Git status update was not being sent to PRs and the PR link in the execution pipeline was incorrect, redirecting back to the same execution link. The PR link redirect was not working for the input expression `<+trigger.payload.pull_req.number>`, so support for this expression has been added. (CI-11759)
+
+#### Harness Platform
```
Comment (bharatgoelharness) [inline release-notes/self-managed-enterprise-edition.md:319]: add this RN in SMP rn as well https://developer.harness.io/release-notes/delegate/#august-2024

File: release-notes/self-managed-enterprise-edition.md [hunk 3]
```diff
@@ -123,6 +123,241 @@ Upon providing your credentials and the release version, the script will proceed
 
 :::
 
+## September 2, 2024, version 0.20.0
```
Comment (bharatgoelharness) [inline release-notes/self-managed-enterprise-edition.md:126]: now this is 3rd september

Comment (GokulBansal0) [reply release-notes/self-managed-enterprise-edition.md]: corrected

**Review by bharatgoelharness — APPROVED** [review]

**Review by avaakash — APPROVED** [review]

**Pull request merged**